mern-ecommerce/server.js

```diff
@@ -44,7 +44,7 @@ app.use(expressValidator());
 app.use(cors());
 
 // routes middleware
-app.use("/api", authRoutes);
+app.use("/api", authRoutess);
```

Comment:
The code snippet has a typo in the route name. It should be `authRoutes` instead of `authRoutess`.